src/test/java/com/salesforce/dynamodbv2/DocGeneratorRunner.java

```diff
@@ -100,6 +100,7 @@
     /*
      * logger -> account
      */
+    @SuppressWarnings("checkstyle:Indentation")
```

Comment:
for when someone (we?) look back at these a while from now and wonder exactly why we suppressed warnings, could you also add a comment (perhaps above the @SuppressWarnings?) akin to "// Checkstyle: 'lambda arguments' has incorrect indentation level 16, expected level should be 12." (or something that mentions it was checkstyle's (apparent) lambda foolishness)? (here and in the other lamba-related locations.)